fedimint-core/src/api.rs

```diff
@@ -613,6 +619,63 @@ impl WsClientConnectInfo {
     }
 }
 
+/// We can represent client connect info as a bech32 string for compactness and
+/// error-checking
+///
+/// Human readable part (HRP) includes the version
+/// [ hrp (4 bytes) ] [ id (48 bytes) ] ([ url len (2 bytes) ] [ url bytes (url
+/// len bytes) ])+
```

Comment:
let's put than in a code block so it's not getting formatted

```rust 

/// ```txt
/// [ hrp (4 bytes) ] [ id (48 bytes) ] ([ url len (2 bytes) ] [ url bytes (url len bytes) ])+
/// ```

```